SVS/model/train.py

```diff
@@ -587,20 +586,14 @@ def train(args):
         elif args.optimizer == "adam":
             out_log += "lr: {:.6f}, ".format(optimizer.param_groups[0]["lr"])
         out_log += "loss: {:.4f}, spec_loss: {:.4f} ".format(
-            train_info["loss"],
-            train_info["spec_loss"],
+            train_info["loss"], train_info["spec_loss"]
```

Comment:
you won't use spec loss for wavernn predictions. So please add a if-else here to determine if the vocoder is used and if you only want to keep the mel loss or still have spec loss